iroh/src/magicsock/relay_actor.rs

```diff
@@ -274,9 +273,24 @@ impl ActiveRelayActor {
     async fn run(mut self) -> anyhow::Result<()> {
         inc!(MagicsockMetrics, num_relay_conns_added);
 
-        loop {
-            let Some(client) = self.run_dialing().instrument(info_span!("dialing")).await else {
-                break;
+        let mut backoff = ExponentialBuilder::new()
+            .with_min_delay(Duration::from_millis(10))
+            .with_max_delay(Duration::from_secs(5))
```

Comment:
I wonder if we should up this now?  I've always found this choice a bit odd, but it is tricky.  Maybe we could set this to 16s?

Reply:
Changed to 16s max